### PR TITLE
Fix `minimize-user-comments`

### PR DESCRIPTION
**Pull request by sharkykh**
Fixes https://github.com/sindresorhus/refined-github/pull/2146#issuecomment-517392427 and https://github.com/sindresorhus/refined-github/pull/2146#issuecomment-517392965

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -16398,6 +16398,11 @@
 					"version": "0.5.2",
 					"resolved": "https://registry.npmjs.org/type-fest/-/type-fest-0.5.2.tgz",
 					"integrity": "sha512-DWkS49EQKVX//Tbupb9TFa19c7+MK1XmzkrZUR8TAktmE/DizXoaoJV6TZ/tSIPXipqNiRI6CyAe7x69Jb6RSw=="
+				},
```
Comment (sharkykh) [inline package-lock.json]: Not sure why this was removed in #2146, but when I `npm install`ed, the lockfile updated.

File: package-lock.json [hunk 2]
```diff
@@ -16398,6 +16398,11 @@
...
+				"webext-detect-page": {
```
Comment (sharkykh) [inline package-lock.json]: Not sure why this was removed in #2146, but when I `npm install`ed, the lockfile updated.

Comment (fregante) [reply package-lock.json]: I'd keep it out for now, it will be updated in #2293

File: source/features/minimize-user-comments.tsx [hunk 3]
```diff
@@ -88,7 +90,7 @@ async function minimizeMutedUserComments(): Promise<void> {
 function init(): void {
 	minimizeMutedUserComments();
 	onNewComments(minimizeMutedUserComments);
-	delegate('.timeline-comment-action', 'click', handleMenuOpening);
+	delegate('summary.timeline-comment-action:not([aria-label="Add your reaction"])', 'click', handleMenuOpening);
```
Comment (fregante) [inline source/features/minimize-user-comments.tsx]: `summary` isn't required here, it's best to leave it out

Comment (sharkykh) [reply source/features/minimize-user-comments.tsx]: Without it, edit buttons get selected too. Is that okay?

**Comment by fregante** [comment]
Thanks @sharkykh! Love your PRs :D

**Pull request merged**